package.json

```diff
@@ -147,6 +147,7 @@
     "html-webpack-plugin": "3.2.0",
     "husky": "1.3.1",
     "identity-obj-proxy": "3.0.0",
+    "intl": "^1.2.5",
```

Comment:
Why do we need this dependency? 🤔

Reply:
It's a devDependency
in tests JSDOM only have `en` 
we use Intl here to have the mock the behaviour of `Number.prototype.toLocaleString`
https://github.com/commercetools/ui-kit/blob/e65a1ec06baca13aecf5f64673c2bf1b9b7e7f0f/src/components/inputs/money-input/money-input.spec.js#L649
https://github.com/commercetools/ui-kit/blob/e65a1ec06baca13aecf5f64673c2bf1b9b7e7f0f/src/components/inputs/money-input/money-input.spec.js#L650

Reply:
I would like to look into doing this without this new dep